diff.py

```diff
@@ -1,47 +1,56 @@
 #!/usr/bin/env python3
 import sys
-import re
-import os
-import ast
-import argparse
-import subprocess
-import difflib
-import string
-import itertools
-import threading
-import queue
-import time
-from typing import Any, Dict, List, NamedTuple, Optional, Set, Tuple, Union
-
 
 def fail(msg):
     print(msg, file=sys.stderr)
     sys.exit(1)
 
-
-MISSING_PREREQUISITES = (
-    "Missing prerequisite python module {}. "
-    "Run `python3 -m pip install --user colorama ansiwrap watchdog python-Levenshtein cxxfilt` to install prerequisites (cxxfilt only needed with --source)."
-)
-
-try:
-    from colorama import Fore, Style, Back  # type: ignore
-    import ansiwrap  # type: ignore
-    import watchdog  # type: ignore
-except ModuleNotFoundError as e:
-    fail(MISSING_PREREQUISITES.format(e.name))
-
 # Prefer to use diff_settings.py from the current working directory
 sys.path.insert(0, ".")
 try:
     import diff_settings
 except ModuleNotFoundError:
     fail("Unable to find diff_settings.py in the same directory.")
+sys.path.pop(0)
 
-# ==== CONFIG ====
+# ==== COMMAND-LINE ====
+
+try:
+    import argcomplete
+except ModuleNotFoundError:
+    argcomplete = None
+import argparse
 
 parser = argparse.ArgumentParser(description="Diff MIPS assembly.")
-parser.add_argument("start", help="Function name or address to start diffing from.")
+
+start_argument = parser.add_argument("start", help="Function name or address to start diffing from.")
+if argcomplete:
+    def complete_symbol(**kwargs):
+        prefix = kwargs['prefix']
```

Comment:
```suggestion
        prefix = kwargs["prefix"]
```
(keeping with [black](https://github.com/psf/black)'s standard of only using " quotes)